backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##support.support.json

```diff
@@ -206,59 +206,67 @@
       }
     },
     "layouts": {
-      "list": [
-        "id",
-        "FullName",
-        "Email",
-        "Phone"
-      ],
       "edit": [
         [
           {
             "name": "FullName",
-            "size": 6
-          },
+            "size": 12
```

Comment:
Could you try to retry sizing them based on User Experience? It could be not so user-friendly to give all of them size 12.